src/transformStateWithClones.js

```diff
@@ -7,7 +7,38 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const resultsOfEachAction = [];
+  const stateCopy = { ...state };
+  const commandAddProperties = 'addProperties';
+  const commandRemoveProperties = 'removeProperties';
+  const commandClear = 'clear';
```

Comment:
```suggestion
  const COMMAND_CLEAR = 'clear';
```
Same with other static variables